pkg/resources/task.go

```diff
@@ -182,7 +182,7 @@ func ReadTask(d *schema.ResourceData, meta interface{}) error {
 		return err
 	}
 
-	if err := d.Set("enabled", task.IsStarted()); err != nil {
+	if err := d.Set("enabled", task.State == sdk.TaskStateStarted); err != nil {
```

Comment:
nit: why we get rid of the helper func from SDK?

Reply:
[Brought back in](https://github.com/Snowflake-Labs/terraform-provider-snowflake/pull/3170)